tests/torchscript/test_model_export.py

```diff
@@ -0,0 +1,403 @@
+import torch
```

Comment:
Read through a couple of Pytorch/Torchscript references:
* https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html
* https://discuss.pytorch.org/t/when-should-i-use-tracing-rather-than-scripting/53883

It seems we should also have tests verifying that we can save/load TorchScript models created via `torch.script.trace`?

Reply:
Done